src/AzureIoTHub.Portal.Server/Controllers/v1.0/EdgeModelsController.cs

```diff
@@ -108,5 +108,15 @@ public virtual async Task<IActionResult> DeleteAvatar(string edgeModelId)
             await this.edgeModelService.DeleteEdgeModelAvatar(edgeModelId);
             return NoContent();
         }
+
+        /*================================================= AWS GreenGrass =====================================================*/
+
+        [HttpPost("aws")]
+        public async Task<IActionResult> CreateGreenGrassDeployment(IoTEdgeModel EdgeModel)
+        {
+            await this.edgeModelService.CreateGreenGrassDeployment(EdgeModel);
+
+            return Ok();
+        }
```

Comment:
Why this method?
You should here passe through the original method and switch between AWS and Azure.